examples/common/chip-app-server/Server.cpp

```diff
@@ -158,11 +159,30 @@ void InitServer(AppDelegate * delegate)
         SuccessOrExit(err = gRendezvousServer.Init(params, &gTransports));
     }
 
+    // TODO: advertise this only when really operational once we support both
+    // operational and commisioning advertising is supported.
+    {
+        constexpr uint64_t kTestFabricId = 5544332211;
+        err                              = Mdns::ServiceAdvertiser::Instance().Advertise(Mdns::OperationalAdvertisingParameters()
```

Comment:
We can still guard with `IsFullyProvisioned` to support advertising `_chipc._udp` for IP rendezvous.

Reply:
Agree. The reason I made it advertise fully here is to always have functional mDNS to test with.

My next PRs (maybe pending some cleanup in usage) would be to add _chipc._udp vs operational advertisiement.